docs/release-notes/8.10.asciidoc

```diff
@@ -1,6 +1,17 @@
 [[release-notes-header-8.10.1]]
 == 8.10
 
+[discrete]
+[[release-notes-8.10.4]]
+=== 8.10.4
+
+[discrete]
+[[bug-fixes-8.10.4]]
+==== Bug fixes
+* Fixes a bug in Timeline that prevented the **Show top _x_** action from showing results ({pull}168339[#168339]).
+* Fixes a bug in Timeline that prevented the **Show top _x_** action from using the Timeline query ({pull}165109[#165109]).
+* Fixes a bug that caused outdated or incorrect data to display on the MITRE ATT&CK® Coverage page ({pull}167917[#167917]). 
```

Comment:
Coverage Overview* page? Not sure how strict we're being with that title in the docs but other than that lgtm!

Reply:
Yeah I guess that's what we call it on the page, I thought we referred to it in docs as the internal name but apparently not

Reply:
Gotcha - thanks for clarifying! I'll resolve this now since we don't need to make further changes to the summary.